Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Action/EditorActionContext.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/ActionManager/Action/EditorActionContext.h>
+
+namespace AzToolsFramework
+{
+    EditorActionContext::EditorActionContext(
+        AZStd::string_view identifier, AZStd::string_view name, AZStd::string_view parentIdentifier, QWidget* widget)
```

Comment:
suggestion: As we're making copies of the strings here, we might want to actually just pass `AZStd::string` here by value and then `AZStd::move` the values into the member variables? Not a huge deal but could be clearer